calculation/calculation.go

```diff
@@ -183,22 +183,73 @@ func isPoolQualified(program types.Program, pool types.Pool, locked uint64) (boo
 	if !atLeastIntegerPercent(locked, pool.TotalLPTokens, program.MinLPIntegerPercent) {
 		return false, fmt.Sprintf("less than %v%% of LP tokens locked", program.MinLPIntegerPercent)
 	}
+	// We'll use a true/false striping to allow/disallow this pool
+	// if it's in the eligible pools, or has an asset that's in the eligible assets, we'll flip it true
+	// if it's in the disqualified pools, or disqualified assets, we'll flip it back false
+	// BUT, if eligible pools and eligible assets are both nil, then we'll assume it's true unless disqualified
+	qualified := program.EligiblePools == nil && program.EligibleAssets == nil && program.EligiblePairs == nil
```

Comment:
Does `EligiblePairs` play into these comments at all? I'm just trying to understand the logic.

Reply:
yea, I added EligiblePairs after writing the comment; I'll update the comment